xtend-contrib/src/main/java/de/oehme/xtend/contrib/Builder.xtend

```diff
@@ -54,6 +67,15 @@ class BuilderProcessor extends AbstractClassProcessor {
 					'''
 					primarySourceElement = cls
 				]
+				addMethod('''set«builtField.simpleName.toFirstUpper»''') [
```

Comment:
Actually, it did work before only because the main function was inside the Person class and it had access to the fields directly. Once the client is external, the original implementation doesn't work anymore.